src/wren_core.c

```diff
@@ -888,6 +906,26 @@ DEF_PRIMITIVE(num_isNan)
   RETURN_BOOL(isnan(AS_NUM(args[0])));
 }
 
+DEF_PRIMITIVE(num_rad)
+{
+  RETURN_NUM(floor(AS_NUM(args[0]) / 57.2957795130823208768));
+}
+
+DEF_PRIMITIVE(num_sign)
+{
+  double value = AS_NUM(args[0]);
+  double sign;
+  if (value > 0) {
```

Comment:
Yep, sorry.

I tend to adhere to the Wren coding-styles but from time to time other projects (and/or personal) style crawls into. :)